web/index.html

```diff
@@ -94,7 +102,7 @@
           <div class="inner"></div>
         </div>
         <div class="view" flex two layout vertical>
-          <div class="header" layout horizontal wrap>
+          <div class="header small" layout horizontal wrap>
             <li><a id="consoletab" selected><span>Console</span></a></li>
```

Comment:
not so sure about this, seems a bit odd that the tab text (Console, Documentation) is smaller than the text in the documentation